Benefite

Olga-Synko opened the pull request.

Olga-Synko committed:
Section Olga
	modified:   index.html

Olga-Synko committed:
Add images
	new file:   images/advertise/Affordable-price.svg
	new file:   images/advertise/Own-production.svg
	new file:   images/advertise/Unsurpassed-taste.svg
	new file:   images/ingredients/cacao.png
	new file:   images/ingredients/coconut-oil.png
	new file:   images/ingredients/milk-chocolate.png
	new file:   images/ingredients/milk.png
	new file:   images/ingredients/nuts.png
	new file:   images/ingredients/rasberry.png
	new file:   images/ingredients/semi-sweet-chocolate.png
	modified:   index.html

Olga-Synko committed:
new file:   images/review/Nazar-Babenko.jpg
	new file:   images/review/Olena-Kampusch.jpg
	new file:   images/review/Victoria-Dobrzhanska.jpg
	modified:   index.html

Lubarskiyk reviewed (changes requested):
1. В секции advertise  видсутний р2
2. Импорт картинок з людми мае тінь  Назар Бабенко 
3. Видсутня стрилочка на карточках ингридіенти

Lubarskiyk committed:
Merge branch 'main' into benefite

Olga-Synko committed:
renamed:    images/advertise/Affordable-price.svg -> images/icons/Affordable-price.svg
	renamed:    images/advertise/Own-production.svg -> images/icons/Own-production.svg
	renamed:    images/advertise/Unsurpassed-taste.svg -> images/icons/Unsurpassed-taste.svg
	new file:   images/icons/arrow.svg
	new file:   images/loved/Nazar-Babenko.jpg
	renamed:    images/review/Olena-Kampusch.jpg -> images/loved/Olena-Kampusch.jpg
	renamed:    images/review/Victoria-Dobrzhanska.jpg -> images/loved/Victoria-Dobrzhanska.jpg
	deleted:    images/review/Nazar-Babenko.jpg

Olga-Synko committed:
Merge branch 'benefite' of https://github.com/Lubarskiyk/chokolat_team into benefite

Lubarskiyk reviewed (approved):
Молодець Оля
